pom.xml

```diff
@@ -119,7 +119,7 @@
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.caconfig.api</artifactId>
-            <version>1.1.0</version>
+            <version>1.1.1</version>
```

Comment:
Isn’t this supposed to be the latest SNAPSHOT? The last release does not yet define „has“

Reply:
Changed this with the latest SNAPSHOT